basket/news/middleware.py

```diff
@@ -1,24 +1,7 @@
-import re
-
 from django.conf import settings
-from django.core.exceptions import MiddlewareNotUsed
-from django.http import HttpResponsePermanentRedirect
-from django.http.request import split_domain_port
 
 from django_statsd.clients import statsd
 from django_statsd.middleware import GraphiteRequestTimingMiddleware
-from mozilla_django_oidc.middleware import SessionRefresh
-
-IP_RE = re.compile(r"^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}$")
-
-
-class OIDCSessionRefreshMiddleware(SessionRefresh):
```

Comment:
I flipped it to remove the custom middleware. Instead of saying "OIDC only the admin", I'm saying "exclude everything but the admin". It's effectively the same but just means we need to exclude new URLs we add. I added a note in the `urls.py` file for that.